src/tests/ftest/harness/core_files.py

```diff
@@ -45,15 +45,16 @@ def test_core_files(self):
             results = run_local(self.log, "cat /proc/sys/kernel/core_pattern", check=True)
         except RunException:
             self.fail("Unable to find local core file pattern")
-        core_path = os.path.split(results.stdout.splitlines()[-1])[0]
-        core_file = "{}/core.gdb.harness.advanced".format(core_path)
-
-        self.log.debug("Creating %s", core_file)
-        try:
-            with open(core_file, "w", encoding="utf-8") as local_core_file:
-                local_core_file.write("THIS IS JUST A TEST\n")
-        except IOError as error:
-            self.fail("Error writing {}: {}".format(local_core_file, str(error)))
+        if not results.stdout.startswith('|'):
```

Comment:
Instead of this should line 45 run the new `scripts/get_core_file_pattern.sh` script?  This makes the test a little less deterministic, as it currently always produces a fake core.gdb file to confirm that launch.py handles it correctly.

Reply:
Good idea!